src/game_initialization/multiplayer.cpp

```diff
@@ -682,8 +682,8 @@ void start_local_game_commandline(CVideo& video, const config& game_config, save
 	}
 
 	std::string label = "";
```

Comment:
And some information: the option seems to be undocumented in the manpage despite having been around since at least 1.10 (commit 2445c2747377408dc69f7b9b5e3b00961b05cfb5 adds it to the boost command-line options).